site/docs/uk/guide/reactions.md

```diff
@@ -0,0 +1,171 @@
+```
+
+Хоча ці два масиви [об'єктів `ReactionType`](https://core.telegram.org/bots/api#reactiontype) технічно надають вам усю інформацію, необхідну для обробки оновлень реакцій, вони все одно можуть бути дещо громіздкими для роботи.
+Ось чому grammY може обчислювати більше корисних речей з оновлення.
+
+### Перевіряємо, як змінилися реакції
+
+Існує [скорочений метод контексту](./context.md#скорочені-методи) під назвою `ctx.reactions`, який дозволяє побачити, як саме змінилася реакція.
+
+Ось як ви можете використовувати `ctx.reactions`, щоб виявити, чи видалив користувач свій голоз "за", але пробачити йому, якщо він все ще зберігає реакцію "ОК".
+
+```ts
+bot.on("message_reaction", async (ctx) => {
+  const { emoji, emojiAdded, emojiRemoved } = ctx.reactions();
+  if (emojiRemoved.includes("👍")) {
+    // Голос "за" було видалено! Неприйнятно.
+    if (emoji.includes("👌")) {
+      // Поки що добре, не караємо.
+      await ctx.reply("I forgive you");
+    } else {
+      // Як вони посміли.
+      await ctx.banAuthor();
+    }
+  }
+});
+```
+
+Метод `ctx.reaction` повертає чотири масиви: додані емодзі, вилучені емодзі, збережені емодзі та список, який показує результат зміни.
+Крім того, є ще чотири масиви для користувацьких емодзі з аналогічною інформацією.
+
+```ts
+const {
+  /** Емодзі, які наразі присутні у реакції цього користувача */
+  emoji,
+  /** Емодзі, які нещодавно додані до реакції цього користувача */
+  emojiAdded,
+  /** Емодзі, які не змінилися після оновлення реакції цього користувача */
+  emojiKept,
+  /** Емодзі, які було видалено з реакції цього користувача */
+  emojiRemoved,
+  /** Користувацьці емодзі, які наразі присутні у реакції цього користувача */
+  customEmoji,
+  /** Користувацьці емодзі, які нещодавно додані до реакції цього користувача */
+  customEmojiAdded,
+  /** Користувацьці емодзі, які не змінилися після оновлення реакції цього користувача */
+  customEmojiKept,
+  /** Користувацьці емодзі, які було видалено з реакції цього користувача */
+  customEmojiRemoved,
+} = ctx.reactions();
+```
+
+Про роботу з оновленнями в приватних і групових чатах було сказано багато.
+Давайте розглянемо канали.
+
+### Обробка оновлень кількості реакцій
+
+У приватних чатах, групах і супергрупах відомо, хто реагує на те чи інше повідомлення.
+Однак для дописів на каналі ми маємо лише список анонімних реакцій.
+Отримати список користувачів, які відреагували на певний допис, неможливо.
+Те ж саме стосується дописів у каналах, які автоматично пересилаються в чати пов'язаних дискусійних груп.
+
+В обох випадках ваш бот отримає оновлення `message_reaction_count`.
+
+Ви можете впоратися з цим так:
+
+```ts
+bot.on("message_reaction_count", async (ctx) => {
+  const counts = ctx.messageReactionCount;
+  // Знову ж таки, ми бачимо лише ідентифікатор повідомлення.
+  const message = counts.message_id;
+  // Ось список реакцій з їхньою кількістю.
+  const { reactions } = count;
```

Comment:
Typo?
```suggestion
  const { reactions } = counts;
```

Reply:
Yes, but this should be corrected in all languages after the merge of this PR. @KnorpelSenf